src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.dao;
+
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {
+    Order add(Order order);
+
+    Order getByUser(User user);
```

Comment:
One user can have multiple Orders.
```suggestion
    List<Order> getByUser(User user);
```